rectools/models/popular.py

```diff
@@ -96,31 +141,37 @@ def __init__(
         inverse: bool = False,
         verbose: int = 0,
     ):
-        super().__init__(verbose=verbose)
-
-        try:
-            self.popularity = Popularity(popularity)
-        except ValueError:
```

Comment:
Why did you removed this check? Same for checks in `PopularInCategory` model

Reply:
Got some troubles with understanding usage of `typing.Literal` for popularity attribute. Brought back all the checks.